features/FEATURE_BLE/targets/TARGET_CORDIO/doc/PortingGuide.md

```diff
@@ -195,70 +174,56 @@ buf_pool_desc_t CordioHCIDriver::get_buffer_pool_description()  {
-should be issued.
-* 2nd page of events mask: Can be achieved by invoking 
-`HciSetEventMaskPage2Cmd((uint8_t *) hciEventMaskPage2)`.
+* **LE event mask:** Call `HciLeSetEventMaskCmd((uint8_t *) hciLeEventMask)`.
 
+* **2nd page of events mask:** Call `HciSetEventMaskPage2Cmd((uint8_t *) hciEventMaskPage2)`.
 
 ###### Stack runtime parameters
 
-Some stack parameters shall be acquired at runtime from the controller: 
-
-* Bluetooth address: Can be queried with `HciReadBdAddrCmd`. Response shall be 
-copied into `hciCoreCb.bdAddr` with `BdaCpy`.
-* Buffer size of the controller: Can be obtained by `HciLeReadBufSizeCmd`. The 
-return parameter *HC_ACL_Data_Packet_Length* shall be copied into 
-`hciCoreCb.bufSize` and the response parameter 
-`HC_Synchronous_Data_Packet_Length`shall be copied into `hciCoreCb.numBufs`. 
-The value of `hciCoreCb.availBufs` shall be initialized with `hciCoreCb.numBufs`.
-* Supported state: Queried with `HciLeReadSupStatesCmd`, the response shall go 
-into `hciCoreCb.leStates`.
-* Whitelist size: Queried with `HciLeReadWhiteListSizeCmd` the response shall go
-into `hciCoreCb.whiteListSize`.
-* LE features supported: Obtained with `HciLeReadLocalSupFeatCmd`. The response 
-shall be stored into `hciCoreCb.leSupFeat`.
-* Resolving list size: Obtained with `hciCoreReadResolvingListSize`. The response
-shall go into `hciCoreCb.resListSize`.
-* Max data length: Acquired with `hciCoreReadMaxDataLen`. The response parameter
-`supportedMaxTxOctets` and `supportedMaxTxTime` shall be pass to the function 
-`HciLeWriteDefDataLen`.
-
-
-The default implementation is: 
-
-```c++
-void HCIDriver::handle_reset_sequence(uint8_t *pMsg) { 
-    // only accept command complete event: 
-    if (*pMsg != HCI_CMD_CMPL_EVT) { 
+At runtime, you can get stack parameters from the controller:
+
+* **Bluetooth address:** Query this with `HciReadBdAddrCmd`. Copy the response into `hciCoreCb.bdAddr` with `BdaCpy`.
+
+* **Buffer size of the controller:** Query this with `HciLeReadBufSizeCmd`.
+  * The return parameter `HC_ACL_Data_Packet_Length` is copied to `hciCoreCb.bufSize`. Copy the response parameter `HC_Synchronous_Data_Packet_Length` into `hciCoreCb.numBufs`. `hciCoreCb.numBufs` initializes the value of `hciCoreCb.availBufs`.
+
+* **Supported state:** Query this with `HciLeReadSupStatesCmd`, and copy the response into `hciCoreCb.leStates`.
+
+* **Whitelist size:** Query this with `HciLeReadWhiteListSizeCmd`, and copy the response into `hciCoreCb.whiteListSize`.
+
+* **LE features supported:** Query this with `HciLeReadLocalSupFeatCmd`, and copy the response into `hciCoreCb.leSupFeat`.
+
+* **Resolving list size:** Query this with `hciCoreReadResolvingListSize`, and copy the response into `hciCoreCb.resListSize`.
+
+* **Max data length:** Query this with `hciCoreReadMaxDataLen`, and pass the response parameters `supportedMaxTxOctets` and `supportedMaxTxTime` to the function `HciLeWriteDefDataLen`.
+
+**Example:**
+
+```
```

Comment:
Please, add the `c++` tag.

Reply:
The c++ tag won't render on our engine anyway; there's no point

Reply:
The doc website is not the only way to interact with the document. 

Developers have to look at github to consult the current version of the document and github renders this tag just fine just like the IDE we use to edit it. 

Also, if at some point syntax highlighting is added to the engine then we will get a better rendering too.

Reply:
True; I don't have an issue with you putting those back in after I've reviewed.